.github/workflows/test.yml

```diff
@@ -1,9 +1,9 @@
 name: CI
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
```

Comment:
We could introduce prettier into our workflows,

[for reference](https://github.com/NEAR-DevHub/neardevhub-bos/blob/84eb5b30a7b56e1261f02036abf02fa93bfd0647/.github/workflows/continuous-integration-workflow.yml#L8)